app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -149,15 +151,24 @@ class AlertFacilityChangeSheet : BaseBottomSheet<
       isFacilitySwitchedPreference.set(false)
       closeSheetWithContinuation()
     } else {
-      router.popWithResult(Succeeded(FacilityChanged()))
+      router.pop()
```

Comment:
I don't remember exactly. I believe it was because I thought we were reading the result of the operation in the screens that open this screen, but it turns out it did not.